src/styles/main.scss

```diff
@@ -0,0 +1,64 @@
+.watch {
+  height: 600px;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  position: relative;
+
+  &__center-circle {
+    background: #2874a6;
+    width: 25px;
+    height: 25px;
+    border-radius: 50%;
+
+    z-index: 3;
+  }
+
+  &__second-hand {
+    background: #2874a6;
```

Comment:
the background property is an individual logic block of styles https://codeguide.academy/html-css.html#css-order